rusk/src/lib/error.rs

```diff
@@ -25,6 +25,8 @@ pub enum Error {
     OutOfGas,
     /// Repeated nullifier in transaction verification
     RepeatingNullifiers(Vec<BlsScalar>),
+    /// Repeated nullifier in the same transaction
+    DoubleNullifiers,
```

Comment:
Maybe "RepeatingNullifiersInTx" ?

Reply:
Yes, I just put the first name...
we can change it later (like the code used for detection, that I really don't like)